R/internal_checks.R

```diff
@@ -0,0 +1,234 @@
+
```

Comment:
This is very much my fault, but in new packages I now try to stick to one function = one file named after that function, mostly because otherwise when I have files like `internals.R` or `auxil.R` or `foo-methods.R` I ended up having to `M-x find-grep-dired` functions I'm looking for.

Reply:
> M-x find-grep-dired

This is sausage to me